src/styles/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 0 auto 40px;
+  width: 160px;
```

Comment:
not crucial, but probably it's better to use margin-bottom and also you can only specify height for an image

Reply:
Thanks, I fixed width and set align-items:center, but when I use margin-bottom only, my image will be large than in mockup